pyrobosim/test/utils/test_pose_utils.py

```diff
@@ -183,26 +205,39 @@ def test_pose_to_from_dict():
     assert quat["z"] == pytest.approx(0.0)
 
 
-def test_construct_pose():
-    """Test pose construct function that accepts various types."""
-    pose_from_list = Pose.construct([1.0, 2.0, 3.0, np.pi / 2.0])
-
+@pytest.mark.parametrize(
+    "angle_units, angle_value",
+    [
+        (
+            "radians",
+            np.pi / 2,
+        ),
+        (
+            "degrees",
+            90.0,
+        ),
+    ],
+)
```

Comment:
is this what the formatter decided was best? Weird. I wonder if removing some of the line breaks will fix things -- I know that sometimes if there used to be line breaks in the old code, the tool won't remove them even if they could actually fit in less lines.

Reply:
Ahh okay, I will fix it.